.github/workflows/publish.yml

```diff
@@ -8,7 +8,100 @@ on:
 jobs:
   tests:
     uses: ./.github/workflows/test.yml
-  publish-release:
+  
+  build-installer:
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - { target: x86_64-unknown-linux-gnu, exe: amd64-linux, os: ubuntu-latest}
```

Comment:
might rename 'exe' to 'suffix' for these for clarity

FYI another way to do this is to use [include](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) -- I have no preference, basically only do this if it gives you better vscode completions and that's useful to you